kratos/includes/define.h

```diff
@@ -36,6 +36,21 @@
     #define KRATOS_COMPILED_IN_WINDOWS
 #endif
 
+// Defining the architecture (see https://sourceforge.net/p/predef/wiki/Architectures/)
+// Check Windows
+#if defined(_WIN32) || defined(_WIN64)
```

Comment:
@loumalouomega in line 35 the `_WIN_64` is not checked...
could this be the reason why #4923 fails?

Reply:
In theory _WIN32 includes _WIN64

https://stackoverflow.com/questions/6679396/should-i-define-both-win32-and-win64-in-64bit-build

    _WIN32: Defined for applications for Win32 and Win64. Always defined.

    _WIN64: Defined for applications for Win64.